page_test.go

```diff
@@ -486,7 +486,7 @@ func TestRepeatedPageTrailingNulls(t *testing.T) {
 	defer reader.Close()
 
 	n, err := reader.ReadRows(rows)
-	if err != io.EOF {
+	if err != nil && err != io.EOF {
```

Comment:
I ran into this as well on a different PR and want to make sure this is desired behavior. Should we handle the EOF internally?

Reply:
This is the desired behavior, it matches the behavior of `io.Reader`, which may not return `io.EOF` if values were read.